paddle/fluid/pybind/pybind.cc

```
@@ -719,6 +733,15 @@ All parameter, weight, gradient are variables in Paddle.
                        [](std::unique_ptr<OpDesc> &p) { return p.release(); });
         return std::make_pair(grad_op_desc_ptrs, grad_to_var);
       });
+  m.def("has_grad_op_maker", [](const std::string op_type) {
+    auto op_info = framework::OpInfoMap::Instance().GetNullable(op_type);
+    return op_info == nullptr ? false : op_info->HasGradOpMaker();
+  });
+  m.def("has_infer_inplace", [](const std::string op_type) {
+    auto op_info = framework::OpInfoMap::Instance().GetNullable(op_type);
```

Why Nullable? Do not hide errors...

---

Done, thanks.